routes/routes.js

```diff
@@ -22,7 +22,7 @@ router.post('/register', async (req, res) => {
         logger.error("Username or password entered does not match pattern required!");
         return res.status(400).send({ error: "Username or password entered does not match pattern required!" });
     }
-    if(fingerprintId === undefined || fingerprintId === null || fingerprintId < 0 || isNaN(fingerprintId) ){
+    if(fingerprintId === undefined || fingerprintId === null || fingerprintId < 0 || isNaN(fingerprintId) || fingerprintId > 162){
```

Comment:
I'm thinking 161. We have 162 slots => 0-161

Reply:
```js
fingerprintId: { type: Number, required: true, max: 162, default: -1 },
```
So this needs to be changed to 161 right

Reply:
yep